.github/workflows/component.yml

```diff
@@ -37,8 +37,6 @@ jobs:
         run: ./gradlew lint --continue 
       - name: Install Frontend Dependencies
         run: npm ci 
-      - name: Run Lint Checks 
-        run: npm run lint 
```

Comment:
This should still be preserved as husky may not be running properly at the client side (or being purposely skipped). GitHub workflow for checking it again to ensure repository consistency is still necessary. "Never Trust a Client"

You need to find a way to lint the diff at workflow, or if not possible, lint everything as before.

Reply:
Thanks for the suggestion! I was also quite worried about husky not running properly on current developers machine as I think that they will have to run `npm install` again for husky to work properly. I think that the husky package should eventually be propagated to all current and new devs as they run `npm install` in their own branches, either when they download new dependencies or when they first set up the repository.

As for now, I am currently looking at using the first solution that I proposed as a way to lint the diffs at workflow, as per what you suggested.